src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -62,10 +62,13 @@
             "No authors found with commits for %s (%s).";
     private static final String MESSAGE_START_ANALYSIS = "Analyzing %s (%s)...";
     private static final String MESSAGE_COMPLETE_ANALYSIS = "Analysis of %s (%s) completed!";
+    private static final String FRAGMENT_NUM_REPOS_ANALYZED = "[%d/%d]";
     private static final String MESSAGE_REPORT_GENERATED = "The report is generated at %s";
     private static final String MESSAGE_BRANCH_DOES_NOT_EXIST = "Branch %s does not exist in %s! Analysis terminated.";
 
     private static Date earliestSinceDate = null;
+    private static int totalNumReposToAnalyze = 0;
+    private static int numReposAnalyzed = 0;
```

Comment:
Don't really like this solution because the values can be easily modified and creates clutters around ReportGenerator.
Can we create a class for this, that is responsible for keeping the count as well as produce the progress message?
Can call it `ProgressTracker` or any better names that you can come up with.